internal/registry/github.go

```diff
@@ -63,3 +64,10 @@ func CreateBrowserCmd(browserURL, os string) (*exec.Cmd, error) {
 		return nil, fmt.Errorf("unsupported platform %s", style.Symbol(os))
 	}
 }
+
+func GetIssueURL(githubURL string) (*url.URL, error) {
```

Comment:
Can we make happy and sad path tests for this? It should be pretty simple, and would help our overall coverage